NineChronicles.Standalone.Executable/AWSSink.cs

```diff
@@ -56,29 +56,39 @@ private async Task Worker(CancellationToken cancellationToken)
         {
             while (!cancellationToken.IsCancellationRequested)
             {
-                while (LogStreamNameGetter is null)
+                try
                 {
-                    await Task.Yield();
-                }
-
-                string logGroupName = LogGroupNameGetter();
-                string logStreamName = LogStreamNameGetter();
-                string sequenceToken = await GetSequenceToken(logGroupName, logStreamName);
-
-                cancellationToken.ThrowIfCancellationRequested();
-                LogEvent logEvent = await _queue.DequeueAsync(cancellationToken);
-                var request = new PutLogEventsRequest(logGroupName, logStreamName, new List<InputLogEvent>
-                {
-                    new InputLogEvent
+                    string logGroupName = LogGroupNameGetter();
+                    string logStreamName = LogStreamNameGetter();
+                    Console.Error.WriteLine($"logGroupName: {logGroupName} / logStreamName: {logStreamName}.");
```

Comment:
This seems a debug print.  Is it intended to be in the commit?

Reply:
Actually, I intended it for later debugging but it is okay to remove if others feel useless.

Reply:
IMO It is ok for debugging later.